base_tier_validation/wizard/comment_wizard.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class CommentWizard(models.TransientModel):
+    _name = 'comment.wizard'
+    _description = 'Comment Wizard'
+
+    res_model = fields.Char(
+        string='Model',
+    )
+    view_id = fields.Many2one(
+        comodel_name='ir.ui.view',
+        string='View',
+    )
+    definition_ids = fields.Many2many(
+        comodel_name='tier.definition',
+    )
+    comment = fields.Char(
+        string='Comment',
+    )
+
+    @api.multi
+    def add_comment(self):
+        self.ensure_one()
+        user_reviews = self.env['tier.review'].search([
+            ('model', '=', self.res_model),
+            ('res_id', '=', self.view_id.id),
+            ('definition_id', 'in', self.definition_ids.ids)
+        ])
+        for user_review in user_reviews:
+            user_review.write({
+                'comment': self.comment,
+            })
+        actions = self.env['ir.actions.act_window'].search(
```

Comment:
You can use self.env.ref('<actoin's external id>') to get action to save multiple lines, can you?